src/dataflow/components/nodes/controls/sensor-select-control.tsx

```diff
@@ -134,13 +134,17 @@ export class SensorSelectControl extends Rete.Control {
       if (!options.length) {
         options.push("none");
       }
-      const titleClass = getChannelString(selectedChannel).includes(kSensorSelectMessage)
+      const channelString = getChannelString(selectedChannel);
+      const titleClass = channelString.includes(kSensorSelectMessage)
                          ? "label unselected"
                          : "label";
+      const topItemClass = channelString.includes("Finding")
```

Comment:
This seems fragile -- relying on  the presence of a user-facing string that this very commit suggests is subject to change. Either it should be a constant like `kSensorSelectMessage `, or there should be some model property that drives both the channel string and the class(es).

Reply:
Fixed.